spec/spec_helper.rb

```diff
@@ -5,6 +5,9 @@
 
 ENV["RAILS_ENV"] = "test"
 
+require "dotenv"
+Dotenv.overload(".sample.env")
```

Comment:
I *think* we need to load the `.env` config here instead, which is created by our setup script.

I'm also curious why we need to do this, I have dockerized other similar rails apps and this wasn't needed AFAIK.